jekyll-assets/_includes/footer.html

```diff
@@ -75,11 +75,11 @@
 }
 #__rptl-footer .__rptl-footer-icon {
   margin-right: 0.5em;
-  fill: #333;
+  fill: var(--subtle-text);
 }
 #__rptl-footer .__rptl-footer-link, #__rptl-footer .__rptl-footer-link--with-icon {
   font-weight: 400;
-  color: #333;
+  color: var(--subtle);
```

Comment:
Is it correct that you've got both `var(--subtle-text);` and `var(--subtle);` ?

Reply:
One is for subtle text. The other is for subtle UI elements :)

I'll unleash a round of comments and CSS organisation today to note some of my trials and errors for the maintainers of the future.